javascript/index.js

```diff
@@ -31,35 +34,66 @@ function printMilliseconds() {
 }
 
 function printSplit() {
-  // ... your code goes here
+  btnRight.addEventListener('click', () => {
```

Comment:
you are already assigning the addEventListener to the buttons in the last part of the code, these will wait for the button to be clicked and then run this function. So in here, the only code needed is the one that prints the result. No need to create a new addEventListener after you click the button.